features/clear_form.feature

```diff
@@ -0,0 +1,246 @@
+@wip
+Feature: Users can reset any form
+
+	As a user who fills out the forms
+	So that I can reset the form
+	I want to add a reset button in each form
+
+Background: Logged in
+	Given the following users exist:
+	  | name            | github_uid      | email                         |
+	  | ESaaS Developer | esaas_developer | esaas_developer@saasbook.info |
+	  | Armando Fox     | armandofox      | armandofox@berkeley.edu       |
+	  | Adnan Hemani    | adnanhemani     | adnanhemani@berkeley.edu      |
+
+	And the following apps exist:
+	  | name                     | description  | org_id | status  |
+	  | AFX Dance                | Awesome!     | 3      | pending |
+	  | Go Bird                  | Great!       | 1      | pending |
+	  | ESaaS Engagement Tracker | The Best App | 2      | pending |
+
+	And the following orgs exist:
+	  | name                         | contact_id |
+	  | Berkeley Student Cooperative | 1          |
+	  | UC Berkeley CS169            | 2          |
+	  | Armando's Secret Society     | 2          |
+	  | Bachelor Party with Adnan    | 3          |
+
+	And the following engagements exist:
+	  | app_id | coaching_org_id | coach_id | team_number | start_date | student_names     |
+	  | 1      | 1               | 1        | 13          | 2012-03-13 | a, b, c           |
+	  | 3      | 2               | 1        | 20          | 2017-09-15 | SC,JD,KH,DK,JS,KP |
+
+	And the following iterations exist:
+	  | engagement_id | end_date   |
+	  | 1             | 2012-05-23 |
+	  | 2             | 2017-11-27 |
+
+	And I am logged in
+
+@javascript
+Scenario: user can reset a form for New App
```

Comment:
Very extensive testing! Make sure to include the Feature ID before you merge it.